client/Player/PlayerSuggestion.ts

```diff
@@ -0,0 +1,39 @@
+module ImprovedInitiative {
+    export class PlayerSuggestion {
```

Comment:
'PlayerViewSuggestor' may be a better name here, as it contrasts well with 'CombatantCommander'